File: Develop/Backend/src/main/java/com/hientran/sohebox/sco/CryptoTokenConfigSCO.java
package com.hientran.sohebox.sco;

import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * @author hientran
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class CryptoTokenConfigSCO extends BaseSCO {

    private static final long serialVersionUID = -68140570073293062L;

    private SearchNumberVO id;

    private SearchTextVO tokenCode;

    private SearchTextVO tokenName;

    private SearchTextVO iconUrl;

    private SearchTextVO nodeUrl;

    private SearchTextVO denom;

    private SearchTextVO addressPrefix;
}

File: Develop/Frontend/src/app/pages/crypto/_sco/cryptoTokenConfigSCO.ts
import { BaseSCO } from "@app/_common/_sco/baseSCO";
import { SearchNumber, SearchText } from "@app/_common/_sco/core_sco";

export class CryptoTokenConfigSCO extends BaseSCO {
  id: SearchNumber;
  tokenCode: SearchText;
  tokenName: SearchText;
  iconUrl: SearchText;
  nodeUrl: SearchText;
  denom: SearchText;
  addressPrefix: SearchText;
}
